<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditExpCurve</class>
 <widget class="QWidget" name="EditExpCurve">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exp Curve</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>To Next Level</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="text_next_lv"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Total</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="text_total"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Basis</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSlider" name="slide_basis">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spin_basis">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Inflation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSlider" name="slide_inflation">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spin_inflation">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>text_next_lv</tabstop>
  <tabstop>slide_basis</tabstop>
  <tabstop>spin_basis</tabstop>
  <tabstop>slide_inflation</tabstop>
  <tabstop>spin_inflation</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>text_total</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>slide_basis</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_basis</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slide_inflation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_inflation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_basis</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slide_basis</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_inflation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slide_inflation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
